params/config.go

```diff
@@ -209,8 +209,6 @@ var (
 	}
 
 	TestRules = TestChainConfig.Rules(0, 0)
-
-	DynamicChainConfigPath string
```

Comment:
Hmm, why not keep it, decode  the full config, and unmarshall the chain part of the config to chain.Config ? I believe this way we'll keep most of the code as is and no need for changing 80 files.
WDTY?